mne/io/brainvision/brainvision.py

```diff
@@ -469,7 +469,10 @@ def _get_eeg_info(vhdr_fname, eog, misc):
         else:
             name, _, resolution, unit = props[:4]
         ch_names[n - 1] = name
-        cals[n - 1] = float(resolution)
+        if len(resolution) > 0:
```

Comment:
Ok test with == "" and your code will be explicit. Also add this comment. And let's merge when Travis is happy